test/parallel/test-tls-psk-circuit.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+const common = require('../common');
+
+if (!common.hasCrypto) {
+  console.log('1..0 # Skipped: missing crypto');
+  return;
+}
```

Comment:
Just fyi, in most tests this looks like this by now:

https://github.com/nodejs/node/blob/d3485121f9d30e482067e751918e7a871bfcf884/test/parallel/test-benchmark-crypto.js#L5-L6